src/RestrictedPython/Guards.py

```diff
@@ -202,10 +202,6 @@ def handler(self, *args):
         return handler
 
     class Wrapper(object):
-        def __len__(self):
-            # Required for slices with negative bounds.
-            return len(self.ob)
-
```

Comment:
I think there should be an entry in the change log mentioning this. So it is easier to find if the code is actually needed.